src/commands/lore/lore.ts

```diff
@@ -0,0 +1,71 @@
+import ICommand from "framework/types/ICommand.ts";
```

Comment:
I prefer prefixing files with their use-case group (like `kagepro_lore`); unless you consider that not having prefixes is more readable ?

Reply:
documented in [101d569](https://github.com/LoganTann/redbeansoup-v2/commit/101d569aa904e01ff85e6130150d5ccd7f1b0e93)